consensus/checks.go

```diff
@@ -56,9 +56,10 @@ func (consensus *Consensus) senderKeySanityChecks(msg *msg_pb.Message, senderKey
 }
 
 func (consensus *Consensus) isRightBlockNumAndViewID(recvMsg *FBFTMessage) bool {
-	if recvMsg.ViewID != consensus.GetCurBlockViewID() || recvMsg.BlockNum != consensus.BlockNum() {
+	blockNum := consensus.getBlockNum()
```

Comment:
What is the difference between `consensus.getBlockNum()` and `consensus.BlockNum()` ?
https://github.com/harmony-one/harmony/blob/258aff0630b9029b35278cfa7ec71a9e232da77e/consensus/consensus.go#L223-L229

Reply:
Naming. Just to be sure no public calls from private methods.